tests/metadataFitsTest.py

```diff
@@ -84,11 +77,9 @@ def test_readInFits(self):
             rootDir = os.path.expanduser(rootDir)
         if not os.path.exists(rootDir):
             print "Data directory {} is required".format(rootDir)
```

Comment:
Tangential, but can we upgrade to `print_function` please? There are a few places in this package still using old style print.